index.html

```diff
@@ -202,47 +177,41 @@ <h2>Photo Gallery</h2>
         </section>
         <section class="survey">
         <h2>Survey</h2>
-         <form >
-         <label>What is your name ?:<br>
-         <input type="text" name="name_field"></label>
```

Comment:
Most of your labels are not using the 'for' attribute as was requested in the lab.
You need an id attribute per form control so that the labels can refer to it using the 'for' attribute
Don't use `<br>` to manage whitespace.  Use CSS!  (This is the reason why the lab asked you to make the form a grid container.)